src/System.Management.Automation/engine/parser/ast.cs

```diff
@@ -7873,10 +7873,16 @@ public class MemberExpressionAst : ExpressionAst, ISupportsAssignment
         /// <param name="static">True if the '::' operator was used, false if '.' is used.
         /// True if the member access is for a static member, using '::', false if accessing a member on an instance using '.'.
         /// </param>
+        /// <param name="genericTypes">The generic type arguments passed to the member.</param>
         /// <exception cref="PSArgumentNullException">
         /// If <paramref name="extent"/>, <paramref name="expression"/>, or <paramref name="member"/> is null.
         /// </exception>
-        public MemberExpressionAst(IScriptExtent extent, ExpressionAst expression, CommandElementAst member, bool @static)
+        public MemberExpressionAst(
+            IScriptExtent extent,
+            ExpressionAst expression,
+            CommandElementAst member,
+            bool @static,
+            IList<ITypeName> genericTypes = null)
```

Comment:
Just saw this -- [we should probably not have default values in public APIs](https://rules.sonarsource.com/csharp/RSPEC-2360)